mozci/console/commands/push.py

```diff
@@ -366,31 +412,12 @@ def handle(self) -> None:
         )
 
         if self.option("recalculate"):
-            try:
-                medium_conf = (
-                    float(self.option("medium-confidence"))
-                    if self.option("medium-confidence")
-                    else 0.8
-                )
-            except ValueError:
-                self.line(
-                    "<error>Provided --medium-confidence should be a float.</error>"
-                )
-                exit(1)
-            try:
-                high_conf = (
-                    float(self.option("high-confidence"))
-                    if self.option("high-confidence")
-                    else 0.9
-                )
-            except ValueError:
-                self.line(
-                    "<error>Provided --high-confidence should be a float.</error>"
-                )
-                exit(1)
-        elif self.option("medium-confidence") or self.option("high-confidence"):
+            classify_parameters = retrieve_classify_parameters(self.option)
+        elif self.option("intermittent-confidence-threshold") or self.option(
```

Comment:
Now that we have more options, we should fail if recalculate isn't set and any of the classify options are used